corehq/apps/api/resources/v0_5.py

```diff
@@ -903,6 +904,40 @@ class Meta(object):
         include_resource_uri = False
 
 
+AnalyticsRoles = namedtuple('AnalyticsRoles', 'domain_name role_names')
+AnalyticsRoles.__new__.__defaults__ = ('', '')
+
+
+class CommCareAnalyticsRolesResource(CorsResourceMixin, Resource):
+    domain_name = fields.CharField(attribute='domain_name', readonly=True)
+    role_names = fields.CharField(attribute='role_names', readonly=True)
+
+    def obj_get_list(self, bundle, **kwargs):
+        request = bundle.request
+        roles_by_domain = get_commcare_analytics_roles_by_user_domains(request.couch_user)
```

Comment:
I can't decide whether or not to add a check here for the SUPERSET_ANALYTICS feature flag.